src/components/usa-details/usa-details.css.js

```diff
@@ -0,0 +1,48 @@
+import { css } from "lit";
+
+export default css`
+  :host {
+    .usa-details {
+        color: #1b1b1b;
+        /* TODO: use var to define this */
+        font-family: Source Sans Pro Web,Helvetica Neue,Helvetica,Roboto,Arial,sans-serif;
+        font-size: 1.06rem;
+        line-height: 1.5;
+        margin-top: .5rem;
+      }
+      .usa-details__summary {
+        /* TODO: use local files */
+        /* TODO: determine if background image is still the best way to add icons here */
+        background-image: url(https://designsystem.digital.gov/assets/img/usa-icons/add.svg);
```

Comment:
**Question (non-blocking):** Is it possible to override this? How would I set my own icon?